package com.example.Arithmetic_try_catch;
 //Exception Handling Without Try and Catch
public class TryCatchArithmeticDemo1 {

	/**
	 * @param args
	 	*/public void division(){
	 		int a = 12;
	 		int b = 2;
	 		int c = a/b;
	 		System.out.println("Division of a and b: "+c);
	 	}
	 	 
	public static void main(String[] args) {
		// TODO Auto-generated method stub
		TryCatchArithmeticDemo1 demo = new TryCatchArithmeticDemo1();
		demo.division();
	}

}
